CosmosRemoteContentMessageConfigurationChangeFeed/handler.ts

```diff
@@ -0,0 +1,53 @@
+import * as E from "fp-ts/lib/Either";
+import { identity, pipe } from "fp-ts/lib/function";
+import * as TE from "fp-ts/lib/TaskEither";
+import * as RA from "fp-ts/ReadonlyArray";
+import { RetrievedRCConfiguration } from "@pagopa/io-functions-commons/dist/src/models/rc_configuration";
+import {
+  UserRCConfiguration,
+  UserRCConfigurationModel,
+  RetrievedUserRCConfiguration
+} from "@pagopa/io-functions-commons/dist/src/models/user_rc_configuration";
+import { NonNegativeInteger } from "@pagopa/ts-commons/lib/numbers";
+import { ILogger } from "../utils/logger";
+import { errorsToError } from "../utils/conversions";
+
+export const handler = (
+  userRCConfigurationModel: UserRCConfigurationModel,
+  logger: ILogger,
+  startTimeFilter: NonNegativeInteger
+) => (
+  documents: ReadonlyArray<unknown>
+): Promise<Error | ReadonlyArray<RetrievedUserRCConfiguration>> =>
```

Comment:
why  `Promise<Error>`? in this way the function will exit with `Success` even if we are returning an error
if we want to mark an invocation as `Failed` we have to throw the error instead of resolving it

Reply:
refactored and throwed errors.